### DISCUSSION TITLE
File size in stand-alone Quarto docs

### DISCUSSION THREAD

**Discussion by smach** [rstudio/gt#1157] [Q&A]
I really like the tables I can produce with {gt} in an html Quarto document! Unfortunately, those documents can get rather large as stand-alone HTML, to the point where I can't use them as email attachments to send to a distribution list at work (> 20M). Not sure if there's anything that can be done to reduce the file size for those?

**Comment by jack-davison** [comment]
Is there any advice on this topic @rich-iannone? I've just updated an embedded-resources Quarto doc to use `DT::datatable()` instead of `{gt}` and it reduced the file size by two-thirds! What could the cause be for `{gt}` to balloon the file size so much? (in my case, these were six, static, 5-line tables with spanner headers and data colours, arranged in 3 rows with a "screen-inset" column width). I'm obviously very ignorant of the internals of how these packages work, but I'd assume a static HTML table would be lighter weight than one with javascript bells and whistles.
